.changeset/deep-mangos-stand.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/datastore": minor
+"__section": feature
+---
+
+Shorter IDs for DataStores and DDSes
+
+Fixed a long pending bug, which now enables us to use shorter ids for Datastores and DDSes when `enableRuntimeIdCompressor:"on"` is set in `IContainerRuntimeOptions`. This change should help in reduction in summary and snapshot sizes as well as improve runtime performance because of smaller memory footprint.
```

Comment:
```suggestion
Fluid Framework will now use shorter IDs for Datastores and DDSes when `enableRuntimeIdCompressor:"on"` is set in `IContainerRuntimeOptions`. This change should help reduce summary and snapshot sizes as well as improve runtime performance because of a smaller memory footprint.
```

Reply:
Done